### ISSUE TITLE
Cannot get index.html

### ISSUE THREAD

**Issue by jeyca** [ether/etherpad-lite#3345]
I forwarded the subdomain pads.example.com to localhost:61273 via:

```
RewriteEngine On
RewriteCond %{HTTPS} !=on
RewriteCond %{ENV:HTTPS} !=on
RewriteRule .* https://%{SERVER_NAME}%{REQUEST_URI} [R=301,L]
RewriteRule (.*) http://localhost:62000/$1 [P]
```
and I can open `pads.example.com/p/mypad`, but when I try to open just `pads.example.com` I get the error "cannot get /index.html". I used the tag/1.6.3 version

**Comment by JohnMcLear** [comment]
Check your rewrite rules.  Closing as Not an Etherpad issue.

**Comment by jeyca** [comment]
exactly, it was a problem because of apache 2.4. One need to set 

    DirectoryIndex disabled

in .htaccess

**Comment by JohnMcLear** [comment]
I love that you are contributing to the software by creating Issues and I'd also super appreciate it if you would close issues like this and leave comments like that without us closing it for you!  Just to save us a few clicks!  Again appreciate you being involved and if you can help out with some other issues I'd really appreciate that..  We're currently sinking dealing with issues like these!

**Comment by jeyca** [comment]
I know, I normally would have done it but just forgot it after I discovered the problem. Sorry for that, it first did not look for me like an apache problem, so I opened up the issue after searching for it in the existing issues.

I am afraid you will always have problems like this in your issue tracker, thats why I am happy that you show your appreciating for contributing. I hope you find a way, still encouraging people to contribute with issues but lighten your work with managing it. 

See it this way: now the next person who will meet the same problem and searches within the closed issues will find the solution himself :) Thats why I contributed my solution here.
